doc/docs/concepts/views.md

```diff
@@ -179,12 +190,14 @@ Using the ViewDefinition editor you can:
 
 ## Output Data Formats
 
-### Conversion to PostgreSQL
+Applying the FHIR ViewDefinition resources to the "base dwh" will generate materialized views which represent a "constrained" set of data to be used for downstream analytics applications (such as dashboards or reporting). 
 
-To be continued... 
+These can be outputted in any tabular format with current support for Database tables and Parquet files.
 
+### Conversion to Database tables
 
-### Conversion to Parquet
+The resulting database tables can be loaded into a commonly used relational database management system such as [PostgresSQL](https://www.postgresql.org/) or [MySQL](https://www.mysql.com/).
```

Comment:
nit: Consider adding a reference to the config param that enables this, i.e., [sinkDbConfigPath](https://github.com/google/fhir-data-pipes/blob/ffecd7c7bf23d86bee0c6dde0dd5d549038f769d/pipelines/controller/config/application.yaml#L173).